node/core/pvf/src/execute/worker.rs

```diff
@@ -235,38 +228,42 @@ impl Response {
 /// The entrypoint that the spawned execute worker should start with. The `socket_path` specifies
 /// the path to the socket used to communicate with the host.
 pub fn worker_entrypoint(socket_path: &str) {
-	worker_event_loop("execute", socket_path, |mut stream| async move {
+	worker_event_loop("execute", socket_path, |stream| async move {
 		let executor = Executor::new().map_err(|e| {
 			io::Error::new(io::ErrorKind::Other, format!("cannot create executor: {}", e))
 		})?;
+		let mutex = Arc::new(Mutex::new((stream, false)));
```

Comment:
`UnixStream` in tokio is not Clone-able and I was forced to put it behind a mutex, hence the small refactor.